app/assets/javascripts/oxalis/controller/scene_controller.js

```diff
@@ -80,6 +85,31 @@ class SceneController {
     this.rootGroup.scale.copy(new THREE.Vector3(...Store.getState().dataset.dataSource.scale));
     // Add scene to the group, all Geometries are then added to group
     this.scene.add(this.rootGroup);
+
+    this.setupDebuggingMethods();
+  }
+
+  setupDebuggingMethods() {
+    // These methods are attached to window, since we would run into circular import errors
+    // otherwise.
+    window.addBucketMesh = (position: Vector3, zoomStep: number, optColor?: string) => {
+      const bucketExtent = constants.BUCKET_WIDTH * 2 ** zoomStep;
+      const bucketSize = [bucketExtent, bucketExtent, bucketExtent];
+      const geo = new THREE.BoxGeometry(...bucketSize);
```

Comment:
I know this is just debugging-code, but still, let's rename it to `boxGeometry` or `helperGeometry` or so :)